style.css

```diff
@@ -0,0 +1,760 @@
+  font-weight: 700;
+  text-align: center;
+  text-decoration-line: underline;
+  text-decoration-color: #ec5242;
+  text-underline-offset: 10px;
+  color: #fff;
+}
+
+.card-stack {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: auto;
+  height: auto;
+  gap: 10px;
+  padding: 0;
+}
+
+.card-stack .cards {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 12px;
+  padding: 5px 10px;
+  background-image: url("images/icebackgr.png");
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.cards-img {
+  box-sizing: border-box;
+  display: flex;
+  width: 44px;
+  height: 44px;
+}
+
+.card-title {
+  display: flex;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 20px;
+  text-align: left;
+  color: #ec5242;
+  width: 50px;
+}
+
+.card-text {
+  font-family: 'Lato', sans-serif;
+  text-align: left;
+  width: 150px;
+  color: #fff;
+  padding-left: 40px;
+}
+
+.view-all {
+  display: none;
+}
+
+.join-us {
+  width: 70%;
+  height: 10%;
+  font-size: 14px;
+  font-weight: 400;
+  color: #fff;
+  margin-top: 5%;
+  padding: 3% 5%;
+  background-color: #ec5242;
+  border-radius: 0.5rem;
+}
+
+.speakers-maintitle {
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-weight: 900;
+  font-style: normal;
+  text-align: center;
+  padding: 20px 10px 0 10px;
+  color: #292929;
+  text-decoration-line: underline;
+  text-decoration-color: #ec5242;
+  text-underline-offset: 10px;
+}
+
+#speaker-cards {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  height: auto;
+  width: auto;
+  margin: 50px;
+}
+
+.each-card {
+  display: flex;
+  gap: 20%;
+  padding: 0;
+}
+
+.speaker-images {
+  width: 65px;
+  height: 35px;
+  z-index: 2;
+}
+
+.speaker-bgimages {
+  width: 15%;
+  height: 15%;
+  z-index: 1;
+  margin-left: -45%;
+}
+
+.card-body {
+  text-align: left;
+  width: 100%;
+}
+
+.speaker-name {
+  text-align: left;
+  font-weight: 600;
+}
+
+.speaker-title {
+  color: #ec5242;
+  text-align: left;
+  font-style: italic;
+  font-weight: 600;
+  text-decoration: underline;
+  text-decoration-color: #d3d3d3;
+  text-underline-offset: 8px;
+  padding-bottom: 4%;
+}
+
+.speaker-bio {
+  text-align: left;
+  width: 100%;
+}
+
+.view-more {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+  margin: 0 10px 30px 20px;
+  padding: 10px 15px;
+  color: #5a5a5a;
+  background-color: transparent;
+  border: 1px solid #d3d3d3;
+  border-radius: 0.3rem;
+  font-family: 'Lato', sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+#chevron {
+  color: #ec5242;
+  padding: 6px 5px 5px 5px;
+}
+
+#footer1,
+#footer2 {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  #dsktop-header {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    height: 15%;
+    width: 100%;
+    z-index: 4;
+  }
```

Comment:
Great job incorporating the suggested changes from the previous reviewer. However, there are still a couple of small issues that need to be addressed:

- [ ] Please remember to adhere to the provided Behance design. As mentioned by the previous reviewer, **you should only display two speakers on the mobile version**. To achieve this, you can add the following class to your CSS file: 🔍

```
#speaker-cards .speaker-cards:nth-child(n+3){ 
  display: none; 
}  
```
Additionally, make sure to include the following code inside your media query:
```
#speaker-cards .speaker-cards:nth-child(n+3){ 
  display: block; 
}
```
- [ ] I noticed that **while you were working on the other requirements required by the previous**, there were some issues with the styling. Specifically, **the image size and the spacing between the image and text** do not align with the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please make the necessary adjustments to ensure that these elements match the design accurately. 😉

  ### Your design:
  ![image](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/107952724/2a25d6b9-aa81-4564-8a28-4dab8876de31)
  ### Behance design:
  ![image](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/107952724/755b0d85-d9f2-4f11-a69d-847eae8ed565)

  _Just a friendly reminder that following the provided template is a key requirement for this project. You're doing a great job so far, and it's important to keep the template in mind as you continue with your design to ensure that your project meets all of the project requirements._
  ![image](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/107952724/d0b9a412-cbf1-4665-b622-cc89958dfb40)